packages/types/src/types/user.ts

```diff
@@ -0,0 +1,10 @@
+import { EmailRecord } from './email-record';
+
+export interface User {
+  username?: string;
+  email?: string;
```

Comment:
is this line accurate?

Reply:
In fact I just copied the types and renamed it

Reply:
Yeah I now but was wondering if this is used or not ^^

Reply:
I think it should not be used no?

Reply:
you mean the email field?

Reply:
email string field

Reply:
I agree, this was on the UserObjectType interface